### ISSUE TITLE
CREST ensemble entropies ~9.1344 J/mol•k

### ISSUE THREAD

**Issue by ghutchis** [grimme-lab/xtb#108]
**Describe the bug**
This would seem to be a bug with CREST rather than xtb, but there's no repo for that, and it might also relate to the GFN2 energy levels.

I've been looking at the computed ensemble entropies from CREST and they seem somewhat high. If I take a simple set of branching alkanes (i.e., adding free rotor CH3 groups)

- methane = 0.0 J/mol•K (**correct** - no conformers, all entropy should be vib / rot / translational)
- ethane = 9.134 J/mol•K ?
- propane = 18.269 J/mol•K (exactly double)
- iso-butane = 27.403 J/mol•K (triple)
- neo-pentane = 36.538 J/mol•K (4x)

**To Reproduce**
Steps to reproduce the behaviour: Run any of the attached xyz files with default CREST options.

[crest-issue.tar.gz](https://github.com/grimme-lab/xtb/files/4172220/crest-issue.tar.gz)

**Expected behaviour**

Since ethane, propane, butane have freely rotatable bonds, shouldn't the entropy scale with the gas constant (i.e., 8.314.. J/mol•K)?

**Comment by ghutchis** [comment]
If there's a physical reason for the ~10% difference, I'd be happy to understand. Ethane, in particular seems as if it should be exactly one rotatable bond, thus 8.314 J/mol•K.

**Comment by fabothch** [comment]
Dear Ghutchis,

thank you for your report/question. 
The value for the ensemble entropy for ethane should be higher than 8.314 J/mol•K!  
The correct approach is described in the Computational Chemistry Comparison and Benchmark DataBase:  https://cccbdb.nist.gov/thermo.asp . There exists a detailed example for calculating S<sub>(free rotor)</sub> of ethane (at 184 K) (almost at the bottom of the page). The formula is:  
![image](https://user-images.githubusercontent.com/49951809/74226683-46e27e80-4cbd-11ea-80bb-89c48d85c0f2.png)
with I<sub>int</sub> being the reduced moment of inertia for the internal rotation, which is geometry dependent!
Their value for S<sub>(free rotor)</sub> of ethane amounts to 10.09 J/mol•K at 184 K. 
We calculate the ensemble entropy using the Gibbs entropy formula:
<a href="https://www.codecogs.com/eqnedit.php?latex=S_{CR}&space;=&space;R&space;\sum^{CRE}_{i=1}&space;p_i\log&space;p_i" target="_blank"><img src="https://latex.codecogs.com/gif.latex?S_{CR}&space;=&space;R&space;\sum^{CRE}_{i=1}&space;p_i\log&space;p_i" title="S_{CR} = R \sum^{CRE}_{i=1} p_i\log p_i" /></a>
because we have information on the rotamers and their population. Using this formula in CREST we end up with a S<sub>(free rotor)</sub> of 9.134 J/mol•K which comes close to the 10.09 J/mol•K. 
In this context, we are going to add a note to the documentation. Using the ensemble entropy from CREST and simply adding it to the RRHO (rigid rotor harmonic oscillator) contribution is not the exact approach! Because in this case 
> the vibrational frequency which is corresponding to the torsion must be deleted if the torsion is treated as a free or hindered rotation! 


For complex systems this is of course normally not practicable since it is unclear which frequencies correspond to torsions and how hindered they actually are.

I hope I could help you!

All the best, 

fabothch

**Comment by ghutchis** [comment]
Right, makes sense - it's been since grad school since I looked at the formula from NIST - I was mentally estimating S_{cr} = R * 1.0 for the Gibbs sum.

I'm asking because we've used CREST to sample CREs across a lot of molecules and have been trying to validate the results in various ways.

I do think a few notes are useful as reminders to readers:
- The CRE entropies are not exactly additive to the RRHO contribution - it's a particular problem because the torsional modes are low frequency and some part of the vibrational entropy. I believe the various MS-T papers by Truhlar cover methods for handling that (e.g doi.org/10.1021/ct3010722)
- Even the RRHO contributions should be evaluated for different conformers, e.g. the rotational contribution can vary for a globular vs. extended conformer.
- An example, e.g. for ethane.
